### PR TITLE
Add new partner logos to homepage

### PR DESCRIPTION
**Pull request by WillPaulRyan**
@RishabhJain2018 

Check my work, let me know if I did everything properly, this is my first real Github contribution.

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@WillPaulRyan Can you please paste a screenshot of it?

**Comment by RishabhJain2018** [comment]
@galipremsagar Please review this PR.

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -201,10 +223,10 @@ <h4 class="w-300 center">Partner Organizations</h4>
                                 </div>
                             </div>
                         </div>
-                        </div>
                     </div>
                 </div>
             </div>
+        </div>
```
Comment (galipremsagar) [inline frontend/src/views/web/landing.html]:
```suggestion
```

Would suggest you remove this tag. This is not something you have introduced, but ideally good to remove this.
I just realised when web storm showed this error of mismatching div.
Screenshot 2019-04-18 at 6 45 54 PM

**Comment by WillPaulRyan** [comment]
@galipremsagar Updated. Nice catch on that other </div> tag.

**Comment by galipremsagar** [comment]
@RishabhJain2018  this can be merged.

**Comment by WillPaulRyan** [comment]
@galipremsagar Okay I've added CSS info for all partner logos

**Comment by WillPaulRyan** [comment]
@galipremsagar @RishabhJain2018 Had some trouble with Git but I think I've updated it all as you asked.
Eval

**Comment by deshraj** [comment]
Hey @WillPaulRyan thanks for the PR. It would be great if you can integrate http://kenwheeler.github.io/slick/ (see 'Responsive Display' one). The current updated list looks cluttered. Would be nice to show the logos in a carousel.

**Comment by galipremsagar** [comment]
@WillPaulRyan , you might want to have a look at triplebyte site as well : https://triplebyte.com/

**Comment by RishabhJain2018** [comment]
Closing this PR due to no activity! Feel free to reopen it.

**Pull request closed without merging**